mango-python/src/mango/QC.py

```diff
@@ -31,28 +31,54 @@ def __init__(self):
         Initializes a Quality Control class.
         """
 
-    def CoverageDistribution(self, coverageRDD, showPlot = True):
+    def CoverageDistribution(self, coverageRDDs, showPlot=True, normalize = False, cummulative = False,):
+        """
+        Computes the coverage distribution of a coverageRDD.
+        :param bdgenomics.adam.rdd.CoverageRDD coverageRDDs: A list of coverageRDDs
+        :param bool showPlot: whether to plot the coverage distribution.
+        :param bool normalize: whether to normalize the distribution.
+        :param bool cummulative: whether to plot the cummulative distribution.
+        :return: list of coverage distributions per input coverageRDD.
+        :rtype: list of list of (string, string) tuples
+        """
+        coverageDistributions = []
+        if (not isinstance(coverageRDDs, list)):
+            coverageRDDs = [coverageRDDs]
+        for coverageRDD in coverageRDDs:
```

Comment:
you want to combine these RDDs first
- combine RDDs (key by some identifier, ie name)
- reduce by key, where key is count and name
- run one collect